packages/plugins/rubygem-recursive-open-struct/rubygem-recursive-open-struct.spec

```
@@ -78,17 +61,21 @@ cp -pa .%{gem_dir}/* \
 
 %files doc
 %doc %{gem_docdir}
-%exclude %{gem_instdir}/.document
+%doc %{gem_instdir}/.document
```

I think we can keep excluding this
```suggestion
%exclude %{gem_instdir}/.document
```